keyboards/thevankeyboards/minivan/keymaps/lexworth/keymap.c

```diff
@@ -0,0 +1,11 @@
+#include QMK_KEYBOARD_H
+
+// Helpful defines
+#define _______ KC_TRNS
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT(KC_GESC, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_BSPC, LT(1,KC_TAB), KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, LT(1,KC_QUOT), KC_LSPO, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSPC, KC_LCTL, KC_LALT, KC_LGUI, KC_ENT, KC_SPC, TG(2), LT(3,KC_APP), KC_DEL),
```

Comment:
A suggestion, but by no means required:

Future edits to your keymap (moving or changing key functions) will be made easier if your layers are formatted to appear visually similar to how the keyboard is laid out, like this:

https://github.com/qmk/qmk_firmware/blob/4a7baaaf50bc9cce3886ff7e1b6455e028d92ab9/keyboards/thevankeyboards/minivan/keymaps/default/keymap.c#L32-L37